src/_scss/pages/search/filters/awardType/awardType.scss

```diff
@@ -3,11 +3,52 @@
     ul.award-types {
         @include unstyled-list;
     }
+    @import '../location/_selectedButtons';
 
-    @import "./_primaryType";
-
-    ul.secondary-award-set {
-        @include unstyled-list;
-        @import "./_secondaryType";
+    .recipient-radio,
+    .checkbox-type-filter {
```

Comment:
Please review the padding on this filter compared to the other filters:
![screen shot 2017-06-01 at 1 33 28 pm](https://cloud.githubusercontent.com/assets/3594363/26692492/ebf4b53e-46ce-11e7-95cf-b6005f665348.png)